src/main/java/com/spotify/docker/client/messages/ContainerConfig.java

```diff
@@ -372,12 +372,28 @@ public Builder volumes(final String... volumes) {
     @JsonProperty("Retries")
     public abstract Integer retries();
 
+    /**
+     * In nanoseconds.
+     */
+    @Nullable
+    @JsonProperty("StartPeriod")
+    public abstract Long startPeriod();
+
     @JsonCreator
     public static Healthcheck create(
             @JsonProperty("Test") final List<String> test,
             @JsonProperty("Interval") final Long interval,
             @JsonProperty("Timeout") final Long timeout,
             @JsonProperty("Retries") final Integer retries) {
+      return create(test, interval, timeout, retries, null);
+    }
+
+    public static Healthcheck create(
```

Comment:
This `create` method, the one with all the parameters, needs to be the `@JsonCreator`. Otherwise `"StartPeriod"` will never be set when creating a `Heathcheck` from json, even if it is set in the json.